src/templates/sitemap.ejs.xml

```diff
@@ -0,0 +1,18 @@
+<urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+<% for (let url of urls ) { %>
+    <url>
+        <loc>
+            https://almanac.httparchive.org/<%= url.loc %>
+        </loc>
+        <lastmod>
+            <%= url.lastmod %>
+        </lastmod>
+        <% for (let alt of url.alternatives ) { %>
+        <xhtml:link 
+            rel="alternate" 
```

Comment:
no need for this because we already handly hreflang tags in the HTML which is recommended.

Reply:
A question: is it unnecessary, or harmful? Do we have to remove it, or can we leave it in both places?

Reply:
Why not both? Can argue don’t need a site map at all as have links on the page but it’s best practice. As is this.

Reply:
It is unnecessary to have both of the methods for hreflangs. Best one is in HTML code. 

> Can argue don’t need a site map at all as have links on the page but it’s best practice. As is this.

Sitemap has another role than the links on page. It's a way to tell Google what are the pages we want to make it discover / index. Also allows as to follow indexing metrics on Search Console.

Reply:
> It is unnecessary to have both of the methods for hreflangs. Best one is in HTML code.

OK but is there any harm in having it as is?

I still go with, if we're going to have a site map, and we're going to have hreflang implemented on our site then surely it's better to use a hreflang sitemap?

Alternatively, if we do not have a hreflang sitemap do we still list all pages, for all languages in the sitemap but just not link them?

If no harm, but no real benefit as we have it in the HTML, then the only reason to remove it, is if we want to reduce code complexity. Will let @mikegeyser make the call on that. Otherwise I'd say keep it, as technically it's the right way to have a multilingual sitemap.

Reply:
OK I've found out the answer to this one:

>I can't see a way to add lastmod dates to alternative languages versions

I've found out is that the sitemap should be reciprocal as well. See [Example in Google Docs](https://support.google.com/webmasters/answer/189077?hl=en). So we should list **both** languages and cross reference them back. Like this:

```
  <url>
    <loc>https://almanac.httparchive.org/en/2019/caching</loc>
    <lastmod>2019-11-05</lastmod>
    <xhtml:link 
               rel="alternate"
               hreflang="de"
               href="https://almanac.httparchive.org/de/2019/caching"/>
    <xhtml:link 
               rel="alternate"
               hreflang="en"
               href="https://almanac.httparchive.org/en/2019/caching"/>
  </url>
  <url>
    <loc>https://almanac.httparchive.org/de/2019/caching</loc>
    <lastmod>2019-11-05</lastmod>
    <xhtml:link 
               rel="alternate"
               hreflang="de"
               href="https://almanac.httparchive.org/de/2019/caching"/>
    <xhtml:link 
               rel="alternate"
               hreflang="en"
               href="https://almanac.httparchive.org/en/2019/caching"/>
  </url>
```

Currently I think we are only listing the English pages with the alternatives and not vice versa (though difficult to tell as no alternative languages currently).

OK for that reason then our Sitemap is going to grow exponentially with each language we add. Plus @mikegeyser said the code [was more complex](https://github.com/HTTPArchive/almanac.httparchive.org/pull/345#issuecomment-551094102):

>@bazzadp The implementation is a lot more complicated, unfortunately

So I think given all this, and the fact we have them listed on the pages themselves, let's revert that last commit and go with the simple sitemap without hreflangs.

Apologies for wasting your time @mikegeyser 😢